File: Tareas/17/Código RUBY/tarea17.rb
=begin
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
=end

print("\nIngrese la magnitud del vector:\n -->")
n = gets().to_i

a = Array.new(n)
suma = 0

for i in(0...n)
	print("Ingresa el valor de vector[#{i}]: ")
	a[i] = gets().to_i
end

print("\nIngrese el valor a buscar:");
b = gets().to_i

for i in(0...n)
	if (a[i]==b)then
		suma = suma + 1
	end
end

puts("Valor a buscar: #{b}")
puts("El numero aparecio #{suma} veces en el vector")
puts("#{a}")

File: Tareas/17/Código JAVA/tarea17.java
/*	
	Nombre: Leslie Estefanía Pérez Cerda
	Matrícula: 1737535
	Hora: Jueves V4-V6
*/

import javax.swing.JOptionPane;
public class tarea17{
	public static void main (String[] args){
		String entrada;
		int i, n, b, suma=0;
		String resultado = " . ";

		entrada = JOptionPane.showInputDialog(null,"Ingrese la magnitud del vector:");
		n = Integer.parseInt(entrada);
		int[] vector = new int[n];

		for(i=0; i<n; i++){
			entrada = JOptionPane.showInputDialog(null,"Ingrese el valor de vector["+i+"]:");
			vector[i] = Integer.parseInt(entrada);
		}

		entrada = JOptionPane.showInputDialog(null,"Ingrese el valor a buscar:");
		b = Integer.parseInt(entrada);

		for(i=0; i<n; i++){
			if(vector[i]==b){ suma = suma + 1; }
		}

		for(i=0; i<n; i++){
			resultado = resultado + vector[i] + " . ";
		}
		JOptionPane.showMessageDialog(null,"Valor a buscar: "+b+"\nEl numero aparecio "+suma+" veces en el vector\n[ "+resultado+" ]");
	} //Fin de función main
} //Fin de clase tarea17